test/idToken.tests.js

```diff
@@ -258,4 +258,20 @@ describe('idToken.validate', function() {
       'Authentication Time (auth_time) claim in the ID token indicates that too much time has passed since the last end-user authentication.'
     );
   });
+  it('should throw when organization is in options, but org_id missing from claim', function() {
+    expectedOptions.organization = 'testorg';
+
+    expect(function() {
+      idToken.validate(generateJWT({ org_id: undefined }), expectedOptions);
+    }).to.throw('Organization Id (org_id) claim must be a string present in the ID token');
+  });
+  it('should throw when org claim doesnt match org expected', function() {
+    expectedOptions.organization = 'testorg';
+
+    expect(function() {
+      idToken.validate(generateJWT({ org_id: 'notExpectedOrg' }), expectedOptions);
+    }).to.throw(
+      'Organization Id (org_id) claim value mismatch in the ID token; expected "testorg", found "notExpectedOrg'
+    );
+  });
```

Comment:
Should we have a test that ensures it doesn't throws when the org_id claim matches the organization from the options?
As well as a test that ensures it doesn't fail when a claim_id is present but the `options.organization` want specified ?